clar2wasm/src/standard/tests.rs

```diff
@@ -821,3 +821,743 @@ fn test_mod_int() {
         )
         .expect_err("expected divide by zero");
 }
+
+#[test]
+fn test_lt_uint() {
+    let (instance, mut store) = load_stdlib().unwrap();
+    let lt = instance.get_func(store.borrow_mut(), "lt-uint").unwrap();
+    let mut result = [Val::I32(0)];
+
+    // 0 < 1 is true
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(0), Val::I64(0), Val::I64(0), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(1));
+
+    // 1 < 0 is false
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(0), Val::I64(1), Val::I64(0), Val::I64(0)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(0));
+
+    // 1 < 1 is false
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(0), Val::I64(1), Val::I64(0), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(0));
+
+    // 1 < 4294967296 is true
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(0), Val::I64(1), Val::I64(1), Val::I64(0)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(1));
+
+    // 1 < 4294967297 is true
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(0), Val::I64(1), Val::I64(1), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(1));
+
+    // 4294967296 < 1 is false
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(1), Val::I64(0), Val::I64(0), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(0));
+
+    // 4294967297 < 1 is false
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(1), Val::I64(1), Val::I64(0), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(0));
+
+    // 4294967296 < 4294967297 is true
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(1), Val::I64(0), Val::I64(1), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(1));
+
+    // 4294967297 < 4294967296 is false
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(1), Val::I64(1), Val::I64(1), Val::I64(0)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(0));
+
+    // 4294967297 < 4294967297 is false
+    lt.call(
+        store.borrow_mut(),
+        &[Val::I64(1), Val::I64(1), Val::I64(1), Val::I64(1)],
+        &mut result,
+    )
+    .expect("call to lt-uint failed");
+    assert_eq!(result[0].i32(), Some(0));
+}
```

Comment:
Can we add one more tests that has a large number which would be negative if interpreted as signed, just to make sure the signedness of the checking is correct? Same comment for the other unsigned ops.